### PR TITLE
ci: update BlackDuck scans

### PR DESCRIPTION
**Pull request by VGervasio**

### CODE REVIEW

File: .github/workflows/security.yml [hunk 1]
```diff
@@ -30,26 +30,52 @@ jobs:
...
 
-      - name: Run Detect Scans
-        run: .github/scripts/blackduck-detect.sh
+      - name: Blackduck Scan
+        uses: lumada-common-services/gh-composite-actions@1.9.0
         env:
-          BLACKDUCK_URL: "${{ secrets.BLACKDUCK_URL }}"
-          BLACKDUCK_TOKEN: "${{ secrets.BLACKDUCK_TOKEN }}"
-          DETECT_VERSION: 8
+          BLACKDUCK_DOCKER_USERNAME: hvservices-service-cicd
```
Comment (VGervasio) [inline .github/workflows/security.yml:74]: Should we create a secret for this value?

**Comment by VGervasio** [comment]
@lumada-design/imperial Please review.

File: .github/workflows/security.yml (hunk 1 above)
Comment (zettca) [reply .github/workflows/security.yml:74]: I don't think it's worth it - it's just a generic CI username.

**Review by zettca — APPROVED** [review]
Looking great 👍 

Just a small note, I think we should convert the `.properties` files into a small script

File: .github/workflows/security.yml (hunk 1 above)
Comment (zettca) [reply .github/workflows/security.yml:74]: I don't think it's worth it - it's just a generic CI username.

File: .github/workflows/security.yml [hunk 2]
```diff
@@ -30,26 +30,52 @@ jobs:
...
+      - name: Get BlackDuck project name
+        run: echo "BLACKDUCK_PROJECT=$(./.github/scripts/getBlackduckProject.sh ${{ matrix.PACKAGE }})" >> $GITHUB_ENV
+
+      - name: Get BlackDuck project version
+        run: echo "PROJECT_VERSION=$( npm pkg get version --workspaces=false | tr -d '"' )" >> $GITHUB_ENV
+        working-directory: packages/${{ matrix.PACKAGE }}
+
+      - name: Load blackduck project properties
+        shell: bash
+        run: echo "BLACKDUCK_ARGS=$(awk '{print}' ORS=' ' .github/security-configurations/blackduck-${{ matrix.PACKAGE }}.properties)" >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (VGervasio) [reply .github/workflows/security.yml]: Suggestion done in new commit.

**Pull request merged**